explainer.md

```diff
@@ -589,14 +593,10 @@ into (regularly) completing a WebAuthn interaction, but it is feasible.
 
 SPC mitigates this attack in the following ways:
 
-1. Requiring a payment policy be set on the iframe, as noted above.
-1. Requiring that the iframe has a user activation.
+1. Requiring a payment policy be set on the iframe, as noted above. This mitigates against a malicious iframe.
+1. Requiring that the iframe has a user activation. This mitigates the case where the top-level frame is colluding. Note: The Working Group is expecting to remove the [requirement](https://w3c.github.io/secure-payment-confirmation/#sctn-payment-extension-registration) that the user agent consume a user activation during registration. For background, see [issue 216](https://github.com/w3c/secure-payment-confirmation/issues/216), including the Chrome Team's security and privacy consideration notes.
```

Comment:
Nope! Not for registration, that's staying. We're proposing to remove it for *authentication*. (As per the issue title).